dapps/digital-twin-detail/src/i18n/en.json

```
@@ -74,10 +74,22 @@
       "transactions-title": "Last Transactions"
     },
     "sharings": {
-      "sharings-title": "Sharings"
+      "filters": {
+        "all": "All",
+        "company": "Companies",
+        "unshared": "Unknown",
+        "user": "Users"
+      },
+      "no-sharings": "This Digital Twin was not shared with any user.",
```

could have been shared in the past and again unshared? 

```suggestion
      "no-sharings": "This Digital Twin is not shared with any user.",
```